app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -61,51 +65,81 @@ const makeURL = (input, isAdmin, projectId) => {
   startUrl += `?${stringify(qsParams, { encode: false })}`;
   return startUrl;
 };
-const makeOptions = (options, projectId) => options.map(option => ({
-  userName: option.full_name || '',
-  userLogin: option.userId,
-  email: option.email || '',
-  disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  userAvatar: getPhoto(option.userId),
-}));
+const makeOptions = (options, projectId) =>
+  options.map((option) => ({
+    userName: option.full_name || '',
+    userLogin: option.userId,
+    email: option.email || '',
+    disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
+    isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
+    userAvatar: getPhoto(option.userId),
+  }));
 const getUsers = (input, isAdmin, projectId) => {
   if (input) {
     const url = makeURL(input, isAdmin, projectId);
-    return fetch(url)
-      .then((response) => {
-        const arr = makeOptions(response.content, projectId);
-        return ({ options: arr });
-      });
+    return fetch(url).then((response) => {
+      const arr = makeOptions(response.content, projectId);
+      return { options: arr };
+    });
```

Comment:
I suppose we should handle a case when a request hasn't been successful

Reply:
+1. Use notifications engine